gantry/commands/logs.py

```diff
@@ -1,39 +1,58 @@
 from typing import Optional
 
 import click
-from beaker import BeakerJob
+from beaker import Beaker, BeakerJob, BeakerTask, BeakerWorkload
 from rich import print
 
 from .. import util
 from ..exceptions import ConfigurationError
 from .main import CLICK_COMMAND_DEFAULTS, main
 
 
+def _get_job(
+    beaker: Beaker, wl: BeakerWorkload, task: BeakerTask, run: Optional[int] = None
+) -> Optional[BeakerJob]:
+    if run is None:
+        return beaker.workload.get_latest_job(wl, task=task)
+    else:
+        # NOTE: ascending sort order on creation time is not implemented server-side yet
+        jobs = list(reversed(list(beaker.job.list(task=task, sort_field="created"))))
+        try:
+            return jobs[run - 1]
+        except IndexError:
+            raise ConfigurationError(f"run number {run} does not exist")
```

Comment:
I wrote a script to do this:
https://gist.github.com/codeviking/3e7a05fae1381ec943210ab8f76cd340

Sometimes there's cases where a distributed task won't have the same number of runs per task (presumably b/c things failed during the setup phase, or b/c there's no synchronization).

I opted in my script to WARN the user about these cases instead of erroring out. Maybe that should be a flag?

```
gantry logs --run 3 --ignore-errors
```

...or something?

Reply:
I think in that case the user probably just wants to see the latest run, in which case they would probably leave `--run` unspecified.